### PR TITLE
added home page

### PR DESCRIPTION
**Pull request by hiteshverma27**
- Added home page 
- Imported CSS from previous ecom submission
- ignore links, will fix them later
[Deploy Preview](https://sweet-melba-16658a.netlify.app/)

### CODE REVIEW

File: src/components/homeMain/HomeMain.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        </div>
+        <div className="catagory-items m-1 p-1 relative">
+
+          <p className="center-div-method-2 w-100per text-align-center text-overlay py-1">Catagory</p>
+          <img className="catagory-img" src="https://github.com/hiteshverma27/LightsOut-Store/blob/development/Assets/wallpaperflare.com_wallpaper%20(1).jpg?raw=true" alt="catagory-img" />
+        </div>
+        <div className="catagory-items m-1 p-1 relative">
+
+          <p className="center-div-method-2 w-100per text-align-center text-overlay py-1">Catagory</p>
+          <img className="catagory-img" src="https://github.com/hiteshverma27/LightsOut-Store/blob/development/Assets/wallpaperflare.com_wallpaper%20(1).jpg?raw=true" alt="catagory-img" />
```
Comment (ranitasahaa1230) [inline src/components/homeMain/HomeMain.js:43]: All these are repetitive codes. Use map function. This will reduce your lines of code

**Review by ranitasahaa1230 — APPROVED** [review]
Your deploy link is not working. Please add a GIF or a screenshot
Reduce your lines of code by separating components or use the map function
Rest looks good

File: src/components/homeMain/HomeMain.js (hunk 1 above)
Comment (hiteshverma27) [reply src/components/homeMain/HomeMain.js:43]: I will rectify that, thanks for pointing out.

**Review by maitrakhatri — APPROVED** [review]
Just one suggestion, use `map` for displaying categories, rest looks good to me

**Pull request merged**